docs/components/connectors/out-of-the-box-connectors/uipath.md

```diff
@@ -0,0 +1,260 @@
+      "content-security-policy":"default-src 'self';script-src 'self' https://orch-cdn.uipath.com https://use.typekit.net/ https://d2c7xlmseob604.cloudfront.net https://platform-cdn.uipath.com https://*.uipath.com https://*.pendo.io;style-src 'self' 'unsafe-inline' https://orch-cdn.uipath.com https://fonts.googleapis.com/css https://use.typekit.net https://p.typekit.net/ https://platform-cdn.uipath.com https://content.usage.uipath.com;img-src 'self' data: https://orch-cdn.uipath.com https://s.gravatar.com https://secure.gravatar.com https://*.wp.com https://*.googleusercontent.com https://i.ytimg.com https://platform-cdn.uipath.com https://*.pendo.io https://*.blob.core.windows.net https://*.amazonaws.com blob:;frame-src 'self' https://*.uipath.com https://*.pendo.io;font-src 'self' https://orch-cdn.uipath.com https://use.typekit.net/ https://fonts.gstatic.com https://platform-cdn.uipath.com data:;connect-src 'self' wss: https://orch-cdn.uipath.com https://primer.typekit.net https://use.typekit.net/ https://sentry.io https://studio-feedback.azure-api.net https://app.launchdarkly.com https://clientstream.launchdarkly.com https://events.launchdarkly.com https://api.smartling.com https://platform-cdn.uipath.com https://*.service.signalr.net https://*.uipath.com https://*.pendo.io https://cloud.uipath.com https://storage.googleapis.com https://*.blob.core.windows.net https://*.amazonaws.com dc.services.visualstudio.com;worker-src 'self' blob:",
+      "x-xss-protection":"1",
+      "x-content-type-options":"nosniff",
+      "x-robots-tag":"noindex,nofollow",
+      "content-type":"application/json; odata.metadata=minimal; odata.streaming=true",
+      "connection":"keep-alive",
+      "cache-control":"no-cache, no-store, must-revalidate",
+      "odata-version":"4.0"
+   },
+   "body":{
+      "@odata.context":"https://cloud.uipath.com/MyOrg/MyTenant/orchestrator_/odata/$metadata#QueueItems",
+      "@odata.count":1,
+      "value":[
+         {
+            "QueueDefinitionId":165001,
+            "Encrypted":false,
+            "Status":"New",
+            "ReviewStatus":"None",
+            "Key":"2196eb07-c96a-4f47-a734-326dd5d58a9d",
+            "Reference":"test",
+            "Priority":"Low",
+            "DeferDate":"2023-01-12T00:00:00Z",
+            "SecondsInPreviousAttempts":0,
+            "RetryNumber":0,
+            "SpecificData":"{\"DynamicProperties\":{\"test\":\"test\"}}",
+            "CreationTime":"2023-01-20T10:13:20.66Z",
+            "RowVersion":"AAAAAE2f4GY=",
+            "OrganizationUnitId":1964413,
+            "OrganizationUnitFullyQualifiedName":"MyCorporateWorkspace",
+            "Id":436141352,
+            "SpecificContent":{
+               "test":"test"
+            }
+         }
+      ]
+   }
+}
+```
+
+## Appendix
+
+### Using UIPath Connector best practice
+
+There is no guarantee a queue item will be processed right away. In that case, we suggest utilizing a BPMN polling loop pattern.
```

Comment:
Can we link to any documentation on this?

Reply:
Added a reference from examples.